server/src/main/webapp/WEB-INF/rails/webpack/views/pages/compare/timeline_modal.tsx

```diff
@@ -123,6 +126,34 @@ export class TimelineModal extends Modal {
     return "Select a pipeline to compare";
   }
 
+  private getPipelineInstanceClassName(maxLength: number) {
+    switch (maxLength) {
+      case 1:
+      case 2:
+        return styles.xSmall;
+      case 3:
+        return styles.small;
+      case 4:
+      case 5:
+        return styles.medium;
+      case 6:
+      case 7:
+        return styles.large;
+      default:
+        return styles.xLarge;
+    }
+  }
+
```

Comment:
Hmm. I am not sure if there is a better way of writing this.

@marques-work -- Thoughts??